runtime/doc/channel.txt

```diff
@@ -20,8 +20,8 @@ There are several ways to open a channel:
 
   2. Through stdin, stdout and stderr of a process spawned by |jobstart()|.
 
-  3. Through the PTY master end of a PTY opened with
-     `jobstart(..., {'pty': v:true})` or |termopen()|.
+  3. Through the PTY master end of a PTY opened with |nvim_open_term()| or
```

Comment:
`nvim_open_term()` does not open a PTY.

Reply:
Thanks, fixed in https://github.com/neovim/neovim/pull/31479